stackslib/src/net/api/tests/postblock_proposal.rs

```diff
@@ -957,3 +968,85 @@ fn replay_validation_test_budget_exceeded() {
         }
     }
 }
+
+#[test]
+#[ignore]
+/// Replay set has [deploy, big_a, big_b]
+/// The block has [deploy, big_a]
```

Comment:
Would it make sense to add another test that checks that the following block, with `[big_b]` is handled correctly as well?